src/core/qgsfileutils.h

```diff
@@ -182,4 +186,49 @@ class CORE_EXPORT QgsFileUtils
 
 };
 
+
+/**
+ * \ingroup core
+ * \class QgsFileSearchTask
+ * \brief Background thread handler for findFile.
+ * \since QGIS 3.18
```

Comment:
Can you rebase your PR and update the "since" part? Thanks

Reply:
Done as requested, thanks!